app.go

```diff
@@ -68,14 +68,18 @@ func main() {
 		fmt.Println(strings.Join(builder, " "))
 		targetCmd := exec.Command(builder[0], builder[1:]...)
 		targetCmd.Dir = tempPath
-		cmdOutput, cmdErr := targetCmd.CombinedOutput()
+		targetCmd.Stdout = os.Stdout
+
+		var stderr bytes.Buffer
+		targetCmd.Stderr = &stderr
+		targetCmd.Start()
+		cmdErr := targetCmd.Wait()
 
 		if cmdErr != nil {
-			fmt.Printf("Error: %s\n" + cmdErr.Error())
+			fmt.Printf("Error: %s\n", cmdErr)
+			fmt.Printf(stderr.String())
```

Comment:
It's not piped, I only set up stdin to do that - I can change it to both if that would be better?